
<launch>
	<node pkg = "drone_architecture" type="NormalVectorCalculator_Node" name="NormalVectorCalculator_subscriber" output="screen"> </node>
    <node pkg = "drone_architecture" type="Uart_Node" name="Uart_publisher" output="screen"> </node>
    <node pkg = "drone_architecture" type="EndEffector_Node" name="EndEffector_Controller" output="screen"> </node>
    <node pkg = "drone_architecture" type="Main_Node" name="Main_Controller" output="screen"> </node>
    <node pkg = "drone_architecture" type="PixhawkInterface_Node" name="Pixhawk_Interface" output="screen"> </node>
    <node pkg = "drone_architecture" type="ControlLoop_Node" name="Control_Loop" output="screen"> </node>
</launch>